packages/backend/src/controllers/authenticationRoles.ts

```diff
@@ -0,0 +1,31 @@
+import {
+    assertUnreachable,
+    OrganizationMemberRole,
+    ProjectMemberRole,
+} from '@lightdash/common';
+
+const inheritedProjectRoleFromOrgRole = (
+    orgRole: OrganizationMemberRole,
+): ProjectMemberRole => {
+    switch (orgRole) {
+        case OrganizationMemberRole.MEMBER:
```

Comment:
Not sure If we need to throw an error or null for this case 🤔 
If the user doesn't have a project role and is only an org member means it doesn't have access to the project and subsequently the space.
Does this mean we need to check the user project role when adding it to a space and also update the space share table when a user is removed from a project or his org role changes? 😕

Reply:
> If the user doesn't have a project role and is only an org member means it doesn't have access to the project and subsequently the space.

I think we currently inherit the role from the org for projects, which means that an admin in the org is an admin in a project by default. The same should be applied here I think..

Reply:
Scenarios: 

A is an org admin
B is an org project
C is an org member

Project X can be accessed by A and B but not by C.

Problem 1:
User A can create a private space and invite C ( the space service should check if invited user can access the project )

Problem 2:

1. User A can create a private space and invite B
2. User A changes B role to Org member, losing access to project X but space access table is still marking user B with access to the space

Reply:
So, from the standup: 

- access roles are not saved in `space shared`but calculated from project (or org if project is missing) 
- member org users should not have access

I'm returning `null` if user does not have project role, and he is member, that should filter out spaces correctly (once enabled) 

I'm not forcing error if someone tries to add an user with `member` role in the backend, I feel that filtering the list of valid users at a given time is something that belongs to the frontend, because the permission is not fixed to the space and it can be changed at any time. But happy to discuss..